### DISCUSSION TITLE
Any suggestion why I get this error on z_isMatch ? Please advise

### DISCUSSION THREAD

**Discussion by iitbhumanish** [zinggAI/zingg#572] [Q&A]
Any suggestion why I get this error on z_isMatch ? Please advise.
Same notebook worked fine in first iteration but started getting this error from second iteration. Now, getting same error even I use same notebook on different data set in first iteration also.

![image](https://user-images.githubusercontent.com/131442577/234001594-e882dd28-627f-430d-82d5-bcd20a5f9fe3.png)

**Comment by iitbhumanish** [comment]
I have  resolved this issue.  Problem was with panda datafram utility to convert into marked parquet file ( matched) over new data set. I have to re-write the code and used Spark-datafram utility to convert into marked parquet file .
trainMatch job also worked fine.

**Comment by sonalgoyal** [comment]
Perfect, glad it worked!

**Reply by iitbhumanish** [reply]
Thanks . please advise on next step or share any document.

I also asked on question 'https://github.com/zinggAI/zingg/discussions/575' . Please advise.
